src/containers/Profile/Profile.jsx

```diff
@@ -1,11 +1,15 @@
-import React from 'react'
+import React from "react";
+import "./Profile.scss";
+import Button from "../../components/Button/Button";
```

Comment:
We want the button to be able to sign the user out -> https://firebase.google.com/docs/auth/web/password-auth#next_steps

This won't need to be tested 👍

Reply:
Sign out from firebase no in place